bundletester/spec.py

```diff
@@ -311,6 +311,8 @@ def filter_yamls(yamls):
 
     result = []
     for yamlfn in yamls:
+        if yamlfn.endswith('metrics.yaml'):
+            continue
```

Comment:
The metrics layer expects to have a metrics.yaml file :( ... meaning right now bundletesting a charm with metrics is failing.